docs/DynamicESDL/DESDL-Mods/ESDLauth_feature.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect2 id="ESDL_Attributes_auth_feature">
+  <title><emphasis role="bold">auth_feature</emphasis></title>
+
+  <para>The auth_feature attribute (valid only for an ESPMethod) allows you to
+  specify means to authenticate user's permission to execute a method.</para>
+
+  <para><emphasis role="bold">Example:</emphasis></para>
+
+  <programlisting> ESPservice MyService
+
+ {
+   ESPmethod MyMethod1(MyMethod1Request, MyMethod1Response);
+   ESPmethod 
+    [
+     description("MyMethod Two"),
+     auth_feature("AllowMyMethod2"),
```

Comment:
Platform defined auth level features are declared in one of the config files (buildset maybe) and the system "automatically" creates the corresponding entry in LDAP. As you point out, for DESDL, the auth_feature would be defined by the user, and the corresponding resouce created by an admin in the security back-end of choice. 
As an illustration of how it would be done for our security back-end of choice (LDAP) @RussWhitehead and/or @mayx might be better equipped to provide details/samples on how to create the resource on the LDAP server.